turbinia/evidence.py

```diff
@@ -527,11 +526,14 @@ def _postprocess(self):
 class DiskPartition(RawDisk):
   """Evidence object for a partition within Disk based evidence.
 
+  More information on dfVFS types:
+  https://dfvfs.readthedocs.io/en/latest/sources/Path-specifications.html
```

Comment:
Thanks for adding this.  I still can't really tell what a "partition location" actually is though :).  It looks like it might be a kind of path, but not really sure what that means since this is outside of a filesystem.  Could we maybe update the description a bit just for that property?